lib/mwUtil.js

```diff
@@ -33,6 +33,15 @@ mwUtil.makeETag = (rev, tid, suffix) => {
  * Normalizes the request.params.title and returns it back
  */
 mwUtil.normalizeTitle = (hyper, req, title) =>  mwUtil.getSiteInfo(hyper, req)
+.catch((e) => {
```

Comment:
Yes, that's what I meant with validated / RB controlled. Will add a note that this is the expectation.